ADOPTERS.md

```diff
@@ -34,3 +38,9 @@ By supporting JSON Schema with a case study, you are documenting its success and
 | Organization | Contact | Description of Use / Reference |
 | --- | --- | --- |
 | GitHub | [Rachael Sewell](https://github.com/rachmari)<br/>[Robert Sese](https://github.com/rsese) | Github JSON Schema case study - [Validating documentation at GitHub](https://json-schema.org/blog/posts/github-case-study) |
+| Cookpad | [Kenshi Shiode](https://github.com/solt9029) | Cookpad JSON Schema case study - [Validating documentation at GitHub](https://github.com/json-schema-org/blog/blob/main/pages/posts/cookpad-case-study-en.md) |
+| Tyler Technology | [Andres Moreno](https://github.com/andmoredev) | Tyler Technology JSON Schema case study - [Validating documentation at GitHub](https://github.com/json-schema-org/blog/blob/main/pages/posts/tyler-technologies-case-study.md) |
+| W3C Web of Things | [Ege Korkan](https://github.com/egekorkan) | W3C Web of Things JSON Schema case study - [Validating documentation at GitHub](https://github.com/json-schema-org/blog/blob/main/pages/posts/w3c-wot-case-study.md) |
+| Remote | [Sandrina Pereira](https://github.com/sandrina-p) | Remote JSON Schema case study - [Validating documentation at GitHub](https://github.com/json-schema-org/blog/blob/main/pages/posts/remote-case-study.md) |
+| Postman | [Juan Cruz Viotti](https://github.com/jviotti) | Postman JSON Schema case study - [Validating documentation at GitHub](https://github.com/json-schema-org/blog/blob/main/pages/posts/postman-case-study.md) |
```

Comment:
The text for the links need to be changed for each case study as they are not all at GitHub 😅. Just an oversight.

Reply:
All those are in github, but I think will be better to use the public link of published on the blog. I'll change it.